src/extensions/correlograms/Correlogram_ReactVis2.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { FunctionComponent } from 'react';
 import { VerticalBarSeries, XAxis, XYPlot, YAxis } from 'react-vis';
 import { createCalculationPool, HitherJobStatusView, useHitherJob } from '../common/hither';
-import { Sorting, SortingSelection, SortingSelectionDispatch } from '../extensionInterface';
+import { mergeGroupForUnitId, Sorting, SortingSelection, SortingSelectionDispatch } from '../extensionInterface';
```

Comment:
Is this something that bled through from the apply-merges-functionality PR?

Reply:
I made a mistake a pushed to the wrong branch. I'll need to remedy this in the merge stage.